uint8 axis 					# Axis type
uint8 AXIS_VERTICAL   = 0
uint8 AXIS_HORIZONTAL = 1
float64 phase      			# Phase
float64 tilt       			# Tilt
float64 gibphase   			# Gibbous phase
float64 gibmag     			# Gibbous magnitude
float64 curve      			# Curve